region/maridia/inner-pink.json

```diff
@@ -5305,7 +5305,7 @@
                     "canIframeSpikeJump",
                     {"spikeHits": 3},
                     {"or": [
-                      "canHorizontalDamageBoost",
```

Comment:
This horizontalDamageBoost refers to the two right side platforms where you boost from one to the other.